spec/sincerely/services/events/aws_ses_event_spec.rb

```diff
@@ -48,10 +48,8 @@
         message_id: 'EXAMPLE7c191be45-e9aedb9a-02f9-4d12-a87d-dd0099a07f8a-000000',
         recipient: 'recipient@example.com',
         timestamp: Time.parse('2017-08-05T00:41:02.669Z'),
-        options: {
-          complaint_feedback_type: 'abuse',
-          user_agent: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.90 Safari/537.36' # rubocop:disable Layout/LineLength
-        }
+        feedback_type: 'abuse',
+        user_agent: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.90 Safari/537.36' # rubocop:disable Layout/LineLength
```

Comment:
Probably you can avoid the `rubocop:disable` here with something like (not tested):
```suggestion
        user_agent: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) ' \
                            'Chrome/60.0.3112.90 Safari/537.36'
```